pkg/comp-functions/functions/vshnmariadb/mariadb_deploy.go

```diff
@@ -528,3 +535,35 @@ func getMariaDBRootPassword(secretName string, svc *runtime.ServiceRuntime) ([]b
 	}
 	return secret.Data["mariadb-root-password"], nil
 }
+
+// updateMariaDBVersionStatus extracts the MariaDB version from the Helm release and updates the status
+func updateMariaDBVersionStatus(comp *vshnv1.VSHNMariaDB, svc *runtime.ServiceRuntime) error {
```

Comment:
Why aren't you just using the value that gets returned by `maintenance.SetReleaseVersion()` at https://github.com/vshn/appcat/blob/develop/pkg/comp-functions/functions/vshnmariadb/mariadb_deploy.go#L120C11-L120C41 ?